content/articles/save-and-load-stacked-ensembles-in-onnx/index.md

```diff
@@ -0,0 +1,256 @@
+# Saving and Loading Stacked Ensemble Classifiers in ONNX format in Python
+
+Stacked ensemble models are learners that increase predictive performance over stand-alone learners by combining the results of two or several models and running them through a meta-learner. The stacked models are different (not a single type), unlike in bagging methods (just decision trees) and each of the model in the stack does not correct the predictions of the previous ones as it happens in boosting. You can learn how to build one on [this](https://www.section.io/engineering-education/ensemble-learning-based-regression-model-using-python/) Sections article by Adhinga Fredrick.
+
+
+Open Neural Network Exchange (ONNX) is an open source format for deep and traditional machine learning developed by Microsoft that has a unified schema for saving models despite the library they were developed in. Launched in December 2017, it gives data scientists and Machine learning engineers a way to persist models without worrying about platform inconsistencies and library version deprecation.
+It is also key as a means to avoid vendor lockin since ONNX models can be deployed on any platform - not just where they were trained.
+Container-based methods of pushing models to production like Docker can also be by-passed altogether.
+
+Saving this type of a model in ONNX format and using it in production can prove challenging as this scenario is not adequately documented.
+So how can stacked ensembles be serialized using ONNX?
+
+## Table of Contents
+  1. Preparing the environments
+  2. Importing and preparing the data
+  3. Building and evaluating the classifier
+  4. Serializing the model to ONNX format
+  5. Loading the model using ONNX runtime Inference Session
```

Comment:
Make these unordered lists using `-`, and indent them towards left.

Add hyperlinks if possible